pkgs/top-level/python-packages.nix

```diff
@@ -6455,6 +6455,10 @@ self: super: with self; {
 
   mohawk = callPackage ../development/python-modules/mohawk { };
 
+  molecule = callPackage ../development/python-modules/molecule { };
```

Comment:
This is also used as a library, not only as an application?

Reply:
Well, upstream lists this:

![image](https://github.com/NixOS/nixpkgs/assets/9713907/4c39409c-78dc-423c-bc00-b30482e00645)

So I would guess yes.

Reply:
Molecule plugins use it as library